[Bug]: SeedFromSlither panics on unparsable Slither constants

### Description

Medusa panics during fuzzer initialization when Slither value seeding is enabled and Slither emits a numeric- or address-typed constant whose value is not a valid base-10 string.

`fuzzing/valuegeneration/value_set_from_slither.go::SeedFromSlither` discards the `ok` return from `big.Int.SetString` in both the integer/uint branch and the address branch:

```go
// integer / uint / int branch
var b, _ = new(big.Int).SetString(constant.Value, 10)
vs.AddInteger(b)
vs.AddInteger(new(big.Int).Neg(b))
vs.AddBytes(b.Bytes())

// address branch
var addressBigInt, _ = new(big.Int).SetString(constant.Value, 10)
vs.AddAddress(common.BigToAddress(addressBigInt))
```

If `SetString` fails it returns `(nil, false)`. The integer branch then panics inside `(*big.Int).Neg`:

```text
panic: runtime error: invalid memory address or nil pointer dereference
math/big.(*Int).Set(...)
math/big.(*Int).Neg(...)
github.com/crytic/medusa/fuzzing/valuegeneration.(*ValueSet).SeedFromSlither
        fuzzing/valuegeneration/value_set_from_slither.go:19
```

The address branch shares the same defect — `common.BigToAddress(nil)` calls `b.Bytes()` on a nil receiver:

```text
panic: runtime error: invalid memory address or nil pointer dereference
math/big.(*Int).Bytes(0x0)
github.com/crytic/medusa-geth/common.BigToAddress
github.com/crytic/medusa/fuzzing/valuegeneration.(*ValueSet).SeedFromSlither
        fuzzing/valuegeneration/value_set_from_slither.go:35
```

This is reachable from real Slither output. `slither/printers/guidance/echidna.py` builds constants via `ConstantValue(str(value), str(value_type))` (lines 219, 223, 245, 251, 271, 281). When `value` is a Python `bytes` object — for example a `bytes32` constant routed through `ConstantFolding` — `str(bytes_obj)` yields the Python repr `b'\x01...'`, which is not a valid base-10 string and trips the integer branch panic. The address branch has the same unchecked parse pattern and panics when an address-typed value is also unparsable. Related upstream context: crytic/slither#2721 ("Slither crashes when using the echidna printer in some bytes32 constants").

### Reproduction Steps

Run `medusa fuzz` with `slither.useSlither = true` against a project whose Slither output includes a non-decimal numeric constant. The fuzzer panics during corpus initialization, before any test executes.

Minimal Go-level reproducer (lives in `fuzzing/valuegeneration/value_set_from_slither_test.go` in the accompanying PR):

```go
func TestSeedFromSlitherInvalidUintDoesNotPanic(t *testing.T) {
    vs := NewValueSet()
    assert.NotPanics(t, func() {
        vs.SeedFromSlither(&compilationTypes.SlitherResults{
            Constants: []compilationTypes.Constant{
                {Type: "uint256", Value: "b'\\x01'"},
            },
        })
    })
}
```

On current `master` this test panics in `SeedFromSlither` at `value_set_from_slither.go:19`. A symmetric `TestSeedFromSlitherInvalidAddressDoesNotPanic` panics at line 35.

### Medusa Version

v1.5.1 (latest release, 2026-03-11). The same code path is on `master` (`3c9412e` at filing time).

### Slither Version

0.11.5

### Crytic-Compile Version

0.3.11

### Operating System

macOS 26.4.1 (Darwin 25.4.0, arm64)

--- a/fuzzing/valuegeneration/value_set_from_slither.go
+++ b/fuzzing/valuegeneration/value_set_from_slither.go
@@ -14,7 +14,10 @@ func (vs *ValueSet) SeedFromSlither(slither *compilationTypes.SlitherResults) {
 	for _, constant := range slither.Constants {
 		// Capture uint/int types
 		if strings.HasPrefix(constant.Type, "uint") || strings.HasPrefix(constant.Type, "int") {
-			var b, _ = new(big.Int).SetString(constant.Value, 10)
+			b, ok := new(big.Int).SetString(constant.Value, 10)
+			if !ok {
+				continue
+			}
 			vs.AddInteger(b)
 			vs.AddInteger(new(big.Int).Neg(b))
 			vs.AddBytes(b.Bytes())
@@ -31,7 +34,10 @@ func (vs *ValueSet) SeedFromSlither(slither *compilationTypes.SlitherResults) {
 			vs.AddBytes([]byte(constant.Value))
 		} else if constant.Type == "address" {
 			// Capture addresses
-			var addressBigInt, _ = new(big.Int).SetString(constant.Value, 10)
+			addressBigInt, ok := new(big.Int).SetString(constant.Value, 10)
+			if !ok {
+				continue
+			}
 			vs.AddAddress(common.BigToAddress(addressBigInt))
 			vs.AddBytes([]byte(constant.Value))
 		}

